### DISCUSSION TITLE
Where is chia_location if I'm using Ubuntu GUI

### DISCUSSION THREAD

**Discussion by Xefier** [swar/Swar-Chia-Plot-Manager#195] [Q&A] [answered]
I'm using chia-blockchain_1.1.4_amd64.deb GUI on Ubuntu, so what would I set as the chia_location?

**Comment by Xefier** [comment, marked as answer]
Oh duh it's still /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/chia
You can delete this if you'd like

**Reply by EclipzzeX** [reply]
I am using Debian 11, and was going to reply with exactly the destination you indicated.  To be fair to you though, it took me a bit to figure this out.  I originally thought it was further down the path.
